extras/indexing/src/main/java/org/apache/rya/indexing/pcj/matching/PCJOptimizer.java

```diff
@@ -90,9 +97,19 @@ public final void setConf(final Configuration conf) {
         if (!init) {
             try {
                 this.conf = conf;
-                this.useOptimal = ConfigUtils.getUseOptimalPCJ(conf);
-                provider = new AccumuloIndexSetProvider(conf);
-            } catch (Exception e) {
+                useOptimal = ConfigUtils.getUseOptimalPCJ(conf);
```

Comment:
Ugh, I hate that we have to do this and we can't use an Interface.  Stupid setConf() init.